maas/plugins/galera_check.py

```diff
@@ -90,6 +90,22 @@ def print_metrics(replica_status):
            replica_status['Threads_connected'], 'connections')
     metric('mysql_max_seen_connections', 'int64',
            replica_status['Max_used_connections'], 'connections')
+    metric('num_of_open_files', 'int64',
+           replica_status['Open_files'], 'files')
+    metric('open_files_limit', 'int64',
+           replica_status['open_files_limit'], 'files')
+    metric('innodb_row_lock_time_avg', 'int64',
+           replica_status['Innodb_row_lock_time_avg'], 'milliseconds')
+    metric('innodb_row_lock_time_max', 'int64',
+           replica_status['Innodb_row_lock_time_max'], 'milliseconds')
+    metric('innodb_deadlocks', 'int64',
+           replica_status['Innodb_deadlocks'], 'deadlocks')
+    metric('access_denied_errors', 'int64',
+           replica_status['Access_denied_errors'], 'access_denied_errors')
+    metric('aborted_clients', 'int64',
+           replica_status['Aborted_clients'], 'aborted_clients')
+    metric('aborted_connects', 'int64',
+           replica_status['Aborted_connects'], 'aborted_connects')
 
```

Comment:
I wonder, with this many metrics, whether we might be best creating a mapping dict, mapping the parameter name obtained from lysql, to the metric name, along with the units.  A bit like here:

https://github.com/rcbops/rpc-openstack/blob/009ae7d2b4751d52d012987a111ad206ba935502/maas/plugins/nova_cloud_stats.py#L32-L60

I'm asking the question, rather than saying we definitely should, just because I'm actually not sure how much better one approach or the other is.